Hailstone.java

```diff
@@ -10,7 +10,23 @@
 
 public class Hailstone extends ConsoleProgram {
```

Comment:
Your solution does not work. When I ran the program it kept printing forever and never stopped.